src/util/timer.cpp

```diff
@@ -12,9 +12,9 @@ void Timer::start() {
     m_time.start();
 }
 
-int Timer::restart(bool report) {
+qint64 Timer::restart(bool report) {
```

Comment:
Since the units are nanoseconds real-world values overflow a signed 32-bit integer pretty fast (2.14 seconds). Correctness matters more than the speed impact using a 64-bit integer on a 32-bit machine has.